Content.Server/Parallax/BiomeSystem.cs

```diff
@@ -59,6 +61,7 @@ public sealed partial class BiomeSystem : SharedBiomeSystem
     private readonly HashSet<EntityUid> _handledEntities = new();
     private const float DefaultLoadRange = 16f;
     private float _loadRange = DefaultLoadRange;
+    private static readonly ProtoId<TagPrototype> MapLoaderTag = "AllowBiomeLoading";
```

Comment:
```suggestion
    private static readonly ProtoId<TagPrototype> AllowBiomeLoadingTag = "AllowBiomeLoading";
```
let's keep it consistent with all the other examples of this being done